src/components/WelcomeBack/WalletList.vue

```diff
@@ -46,17 +125,39 @@ export default {
 @import '@/styles/_colors.scss';
 
 .wallet-list {
+  min-width: 500px;
   .text {
     margin-bottom: 32px;
+    &.header {
+      font-size: 20px;
+    }
   }
-
   .list {
-    height: 240px;
-
+    .row {
+      margin-bottom: 16px;
+      &:last-of-type {
+        margin-bottom: 0px;
+      }
+      .error {
+        margin-top: 8px;
+        color: $red-3;
+      }
+    }
     .wallets {
-      overflow-y: auto;
-      height: 200px;
+      // overflow-y: auto;
+      // height: 200px;
       margin-bottom: 24px;
+      .wallet {
+        margin-bottom: 16px;
+        &:last-of-type {
+          margin-bottom: 0px;
+        }
+      }
+      .el-input {
```

Comment:
Can you remove this style? Is this being overwritten in the element configuration file?